sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -783,10 +786,16 @@ void CAFMaker::produce(art::Event& evt) noexcept {
     // corresponding to a neutrino) that could plausibly be reweighted. This
     // avoids the need for special configuration for cosmics or single particle
     // simulation, and real data.
-    if(!fmpewm && mctruth->NeutrinoSet()) fmpewm = FindManyPStrict<sbn::evwgh::EventWeightMap>(mctruths, evt, fParams.SystWeightLabel());
+    if(fmpewm.empty() && mctruth->NeutrinoSet()){
```

Comment:
Should this check be here? There might be systematic uncertainties for generators other than neutrinos. 

At least, this may be something to revisit in the future.

Reply:
This is like this due to previous complaints about errors running CAFMaker over non-neutrino MC with the reweight label still set. I think we actually need to have a proper conversation about what the convention is for set/unset product labels and existing/missing products before we can know what the right answer is here.

Reply:
Ok! In any case this can be deferred for longer term I think.